ENG-2988 Update Jetty to 10.0.7

cecchisandrone opened the pull request.

Commit:
Updated Jetty to 11.0.7

entando-jx commented:
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **entando-jx**

If they are not already assigned, you can assign the PR to them by writing `/assign @entando-jx` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-docker-base-images).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-docker-base-images/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":["entando-jx"]} -->

Commit:
Changed to jetty 10 and fixed jetty.xml

Commit:
Fixes

- Fix to missing Jetty logs
- Removed war config from jetty as not working in jetty 10.0.7

Commit:
Updated eap base image

entando-jx commented:
@cecchisandrone: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | 7da6636e0f1fa9f2e7e04fcb602551805e22d7bb | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

entando-jx commented:
@cecchisandrone: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | 7709077a3a3d96b5b73ce043f12670a50e770555 | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

entando-jx commented:
@cecchisandrone: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | 0ec37d199e9b2502addaf8ec28fdb86a68cdaaf2 | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

entando-jx commented:
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/entando-docker-base-images/pull/19#pullrequestreview-832739679" title="Approved">BrenoQVDS</a>*
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **entando-jx**

If they are not already assigned, you can assign the PR to them by writing `/assign @entando-jx` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-docker-base-images).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-docker-base-images/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":["entando-jx"]} -->

cecchisandrone commented:
/retest

entando-jx commented:
@cecchisandrone: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | 0ec37d199e9b2502addaf8ec28fdb86a68cdaaf2 | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->
